.fluidlintallrc-custom-options.json

```diff
@@ -77,5 +77,8 @@
         "options": {
             "configFile": "@expand:fluid.module.resolvePath(%fluid-lint-all/.stylelintrc-custom.json)"
         }
+    },
+    "yaml": {
+        "excludes": ["tests/fixtures/yaml/bad.yml"]
```

Comment:
This seems incorrect, we should be passing configuration options to avoid the error that would otherwise be reported, not disabling the check.  I'll update this.

Reply:
There aren't many configurable options, but there are enough that I was able to add support and tests.